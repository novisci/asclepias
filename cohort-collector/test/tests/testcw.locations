test/tests/testcw1.json
test/tests/testcw2.json
test/tests/testcw3.json
