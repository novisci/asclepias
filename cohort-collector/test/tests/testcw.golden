{"example":[{"attritionInfo":[[{"tag":"SubjectHasNoIndex"},0],[{"contents":[1,"dummy"],"tag":"ExcludedBy"},0],[{"tag":"Included"},6]],"totalUnitsProcessed":6,"totalSubjectsProcessed":6},{"contents":{"ids":[["a",["2010-07-06","2010-07-07"]],["b",["2010-07-06","2010-07-07"]],["d",["2010-07-06","2010-07-07"]],["e",["2010-07-06","2010-07-07"]],["f",["2010-07-06","2010-07-07"]],["g",["2010-07-06","2010-07-07"]]],"cohortData":[[5,5,5,5,5,5],[true,true,true,true,true,true]],"attributes":[{"name":"myVar1","type":"Count","attrs":{"getShortLabel":"somelabel","getPurpose":{"getRole":["Outcome"],"getTags":[]},"getLongLabel":"another label","getDerivation":""}},{"name":"myVar2","type":"Bool","attrs":{"getShortLabel":"","getPurpose":{"getRole":[],"getTags":[]},"getLongLabel":"","getDerivation":""}}]},"tag":"CW"}]}